src/scanners/rdf.js

```diff
@@ -24,7 +24,7 @@ export default class RDFScanner extends BaseScanner {
         },
       }).parseFromString(this.contents, 'text/xml');
 
-      resolve(xmlDoc);
+      resolve(xmlDoc, null, {namespace: this.namespace});
```

Comment:
We do have the filename available via `this.filename` here (it's passed into all classes that extend `BaseScanner`) so this should be:

```js
        resolve(xmlDoc, this.filename, {namespace: this.namespace});
```